### DISCUSSION TITLE
How do insert an mp4 video into the report?

### DISCUSSION THREAD

**Discussion by szabo707** [quarto-dev/quarto-cli#8101] [Q&A]
### Description

Can you help me with inserting an .mp4 video into the report? I've tried the following methods, and none of them seem to work.

```
{=html}
<video controls>
  <source src="video.mp4" type="video/mp4">
</video>
```

``` 
{{< video video.mp4 >}}
```

Thank you.

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/authoring/videos.html>.

> I've tried the following methods, and none of them seem to work.

I'm afraid this does not help us understand the issue.
Did you try using the latest 1.4 pre-release? If yes, please provide a self-contained reproducible example, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Reply by szabo707** [reply]
Im using quarto 1.3.433 and I trying everything from https://quarto.org/docs/authoring/videos.html :(

**Reply by mcanouil** [reply]
1. 1.3.433 is not the latest stable version
2. Quoting myself: "Did you try using the latest 1.4 pre-release?"

**Reply by szabo707** [reply]
Not possible, as we use a shared VM.

**Reply by mcanouil** [reply]
So I am afraid your are stuck until you upgrade.

The issue was fixed [v1.4.185](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.4.185) (https://github.com/quarto-dev/quarto-cli/issues/5496)
